src/processor/operator/persistent/reader/rdf/rdf_scan.cpp

```diff
@@ -203,6 +203,50 @@ static std::unique_ptr<TableFuncSharedState> RdfAllTripleScanInitSharedState(
         bindData->config.copy(), std::move(rdfConfig), bindData->store);
 }
 
+static double RdfResourceInMemScanProgressFunc(TableFuncSharedState* sharedState) {
```

Comment:
For `RdfResourceInMemScan` we actually scans both `rtStore` and `ltStore`. So change size to `rtSize + ltSize` and cursor to `rtCursor + ltCursor`